packages/hoppscotch-app/helpers/curl/sub_helpers/contentParser.ts

```diff
@@ -158,14 +158,12 @@ const getXMLBody = (rawData: string) =>
     O.alt(() => O.some(rawData))
   )
 
-const getFormattedJSON = (jsonString: string) =>
-  pipe(
-    jsonString.replaceAll('\\"', '"'),
-    safeParseJSON,
-    O.map((parsedJSON) => JSON.stringify(parsedJSON, null, 2)),
-    O.getOrElse(() => "{ }"),
-    O.of
-  )
+const getFormattedJSON = flow(
+  safeParseJSON,
+  O.map((parsedJSON) => JSON.stringify(parsedJSON, null, 2)),
+  O.getOrElse(() => "{ }"),
+  O.of
+)
```

Comment:
Why wrap this in Option if it is never going to be having a None state ?

Reply:
I thought it would be nice that content type remained application/json since the parser did not display any errors. A subtle hint.

Reply:
Umm okay, I will allow for now because this is just a minor thing.
But this is an anti-pattern and should not be repeated.

Reply:
hmm